downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/GenerateGOAnnotationFile/GOAGeneratorUtilities.java

```diff
@@ -178,34 +176,38 @@ public static Integer assignDateForGOALine(GKInstance entityInst, String goaLine
 
     /**
      * Retrieves date from instance and formats it for GO annotation file.
-     * @param instanceEditInst
-     * @return
-     * @throws Exception
+     * @param instanceEditInst -- GKInstance, instanceEdit from either a Modified or Created instance.
+     * @return -- Integer, from instanceEdit's dateTime. Parsed to remove the Timestamp.
+     * @throws Exception -- MySQLAdaptor exception.
      */
     private static int getDate(GKInstance instanceEditInst) throws Exception {
         return Integer.valueOf(instanceEditInst.getAttributeValue(ReactomeJavaConstants.dateTime).toString().split(" ")[0].replaceAll("-", ""));
     }
 
     /**
      * Iterates through the lines in the 'goaLines' list, retrieves the date associated with that line and also adds the 'Reactome' column before adding it to the gene_association.reactome file.
-     * @param filename
-     * @throws IOException
+     * @throws IOException -- File writing/reading exceptions.
      */
-    public static void outputGOAFile(String filename) throws IOException {
+    public static void outputGOAFile() throws IOException {
 
-        Files.deleteIfExists(Paths.get(filename));
+        Files.deleteIfExists(Paths.get(GOAGeneratorConstants.GOA_FILENAME));
         List<String> sortedGoaLines = new ArrayList<>(goaLines);
         Collections.sort(sortedGoaLines);
-        BufferedWriter br = new BufferedWriter((new FileWriter(filename)));
+        BufferedWriter br = new BufferedWriter((new FileWriter(GOAGeneratorConstants.GOA_FILENAME)));
         br.write("!gaf-version: 2.1\n");
         for (String goaLine : sortedGoaLines) {
-            br.append(goaLine + "\t" + dates.get(goaLine) + "\tReactome\t\t\n");
+            br.append(goaLine + "\t" + dates.get(goaLine) + "\t" + GOAGeneratorConstants.REACTOME_STRING + "\t\t\n");
         }
         br.close();
     }
 
-    // Move file into DownloadDirectory folder corresponding to release number.
-    public static void moveFile(String filename, String targetDirectory) throws IOException {
-        Files.move(Paths.get(filename), Paths.get(targetDirectory), StandardCopyOption.REPLACE_EXISTING);
+    /**
+     * Move file into DownloadDirectory folder corresponding to release number.
+     * @param targetDirectory -- String, where the file will be moved to.
+     * @throws IOException -- If file or targetDirectory do not exist, this will be thrown.
+     */
+    public static void moveFile(String targetDirectory) throws IOException {
+        targetDirectory += GOAGeneratorConstants.GOA_FILENAME;
```

Comment:
It's usually best to avoid modifying a parameter.  Could a new variable be created for target file path?